MAUI/NumericEntry/Migration.md

```diff
@@ -0,0 +1,83 @@
+---
+layout: post
+title: Migrating from Xamarin to .NET MAUI SfNumericEntry | Syncfusion 
+description: Learn about Migrating from Syncfusion Xamarin NumericTextBox and NumericUpDown to Syncfusion .NET MAUI NumericEntry control and more.
+platform: maui
+control: SfNumericEntry
+documentation: ug
+---  
+
+# Migrating from Xamarin SfNumericTextBox and SfNumericUpDown to .NET MAUI SfNumericEntry
+
+To migrate easier from the Xamarin SfNumericTextBox and SfNumericUpDown to `.NET MAUI SfNumericEntry`, most of the APIs are kept from the Xamarin SfNumericTextBox and SfNumericUpDown  in MAUI SfNumericEntry. However, to maintain the consistency of API naming in MAUI SfNumericEntry, rename some of the APIs. The APIs changed in MAUI SfNumericEntry from Xamarin SfNumericTextBox and SfNumericUpDown are detailed as follows.
+
+## Namespaces 
+
+<table>
+<tr>
+<th>Xamarin SfNumericTextBox <br> Xamarin SfNumericUpDown</th>
+<th>.NET MAUI SfNumericEntry</th></tr>
+<tr>
+<td>Syncfusion.SfNumericTextBox.XForms</td>
+<td>Syncfusion.Maui.Inputs</td></tr>
+</table>
+
+## Properties
+
+<table> 
+<tr>
+<th>Xamarin SfNumericTextBox <br> Xamarin SfNumericUpDown </th>
+<th>.NET MAUI SfNumericEntry</th>
+<th>Description</th></tr>
+<tr>
+<td>[BorderColor](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_BorderColor)</td>
+<td>[Stroke](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_Stroke)</td>
+<td>Gets or sets a color that describes the stroke.</td></tr>
+<tr>
+<td>[AllowDefaultDecimalDigits](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_AllowDefaultDecimalDigits)</td>
+<td>[AllowNull](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_AllowNull)</td>
+<td>Gets or sets a value indicating whether the SfNumericEntry allows null value or not.</td></tr>
+<tr>
+<td>[ClearButtonVisibility](https://help.syncfusion.com/cr/xamarin/Syncfusion.SfNumericTextBox.XForms.SfNumericTextBox.html#Syncfusion_SfNumericTextBox_XForms_SfNumericTextBox_ClearButtonVisibility)</td>
+<td>[ShowClearButton](https://help.syncfusion.com/cr/maui/Syncfusion.Maui.Inputs.SfNumericEntry.html#Syncfusion_Maui_Inputs_SfNumericEntry_ShowClearButton)</td>
+<td>Gets or sets a value indicating whether to show or hide the clear button in SfNumericEntry.</td></tr>
```

Comment:
<td>Get or set a value indicating whether to show or hide the clear button in the SfNumericEntry.</td></tr>